### ISSUE TITLE
self-update?

### ISSUE THREAD

**Issue by stoplion** [yarnpkg/website#1058]
Hi..
Just wondering what is the actual purpose of https://classic.yarnpkg.com/en/docs/cli/self-update/?

It always shows up in Google's search pages. And the page only informs me that `self-`update is not available.

**Comment by nhooyr** [comment]
It must have been a command that used to work but is now deprecated.

**Comment by Daniel15** [comment]
Yeah, this was a command available in the original Yarn release, but it was removed at some point.
